bin/gcli.sh.template

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Referred from Apache Submarine's common.sh implementation
+# bin/common.sh
+
+set -e
+
+if [ -L "${BASH_SOURCE-$0}" ]; then
+    BIN_DIR=$(dirname "$(readlink "${BASH_SOURCE-$0}")")
+else
+    BIN_DIR=$(dirname "${BASH_SOURCE-$0}")
+fi
+
+BIN_DIR=$(
+    cd "$BIN_DIR" || exit 2
+    pwd
+)
+
+. "${BIN_DIR}/common.sh"
+
+AUXLIB_DIR="${GRAVITINO_HOME}/auxlib"
+
+# Search strategy for finding the Gravitino CLI JAR:
+# 1. First try to find the development snapshot version in auxlib
+CLI_JAR=$(find "${AUXLIB_DIR}" -name "gravitino-cli-*-incubating-SNAPSHOT.jar" 2>/dev/null)
+# 2. If not found, try to find the release version (without SNAPSHOT)
+if [ -z "$CLI_JAR" ]; then
+    CLI_JAR=$(find "${AUXLIB_DIR}" -name "gravitino-cli-*-incubating.jar" 2>/dev/null)
+fi
+
+# 3. Last resort: try to find any CLI JAR with any suffix pattern
+if [ -z "$CLI_JAR" ]; then
+    CLI_JAR=$(find "${AUXLIB_DIR}" -name "gravitino-cli-*.jar" 2>/dev/null)
+fi
+
+# 4. If still not found, exit with an error
+if [ -z "$CLI_JAR" ]; then
+    echo "ERROR: Could not find CLI jar in ${AUXLIB_DIR}"
+    exit 1
+fi
+
+VERSION=$(basename "$CLI_JAR" | grep -o '[0-9]\+\.[0-9]\+\.[0-9]\+')
+if [ -z "$VERSION" ]; then
+    echo "WARNING: Could not extract version from JAR filename: $(basename "$CLI_JAR")"
+else
+    echo "Find CLI jar version: $VERSION"
+fi
```

Comment:
I don't think we need to echo anything, just run the java command should be fine.